test/dommock/DOMMockBuilder.ts

```diff
@@ -1084,6 +1085,15 @@ export default class DOMMockBuilder {
 
       disconnect(): void {}
     };
+
+    GlobalAny.crypto = {
+      getRandomValues(_data: Uint32Array): Uint32Array {
+        const num = Long.fromNumber(Math.trunc(Math.random() * Number.MAX_SAFE_INTEGER));
+        // Add (2^64 - 2^53) / 2. Shift 53 bit range to middle of 64 bit range.
+        num.add(9218868437227405312);
+        return new Uint32Array([num.high, num.low]);
```

Comment:
Rather than biasing, I would suggest just calling random twice for each 32-bit part:

```
return new Uint32Array([Math.trunc(Math.random() * 0x100000000), Math.trunc(Math.random() * 0x100000000)]);
```